packages/sdk/src/AbstractConfigManager.ts

```diff
@@ -0,0 +1,654 @@
+                    label: name!,
+                    description: profile!.host!,
+                })),
+                {
+                    label: "Migrate From SSH Config",
+                    separator: true,
+                },
+                ...this.filteredMigratedConfigs.map(({ name, hostname }) => ({
+                    label: name!,
+                    description: hostname,
+                })),
+            ],
+            placeholder: "Select configured SSH host or enter user@host",
+        });
+
+        // If nothing selected, return
+        if (!result) return;
+
+        // If result is add new SSH host then create a new config, if not use migrated configs
+        this.selectedProfile = this.filteredMigratedConfigs.find(
+            ({ name, hostname }) => result?.label === name && result?.description === hostname,
+        );
+
+        if (result.description === "Custom SSH Host") {
+            const createNewConfig = await this.createNewConfig(result.label);
+            if (!createNewConfig) return undefined;
+            this.selectedProfile = createNewConfig;
+        } else if (result.label === "$(plus) Add New SSH Host...") {
+            const createNewConfig = await this.createNewConfig();
+            if (!createNewConfig) return undefined;
+            this.selectedProfile = createNewConfig;
+        }
+
+        // If an existing team config profile was selected
+        if (!this.selectedProfile) {
+            const foundProfile = this.sshProfiles.find(({ name }) => name === result.label);
+            if (foundProfile) {
+                const validConfig = await this.validateConfig({
+                    hostname: foundProfile?.profile?.host,
+                    port: foundProfile?.profile?.port,
+                    privateKey: foundProfile?.profile?.privateKey,
+                    keyPassphrase: foundProfile?.profile?.keyPassphrase,
+                    user: foundProfile?.profile?.user,
+                    password: foundProfile?.profile?.password,
+                });
+                if (validConfig === undefined) return;
+
+                await this.setProfile(validConfig, foundProfile.name);
+                return { ...foundProfile, profile: { ...foundProfile.profile, ...validConfig } };
+            }
+        }
+
+        // Current directory open in vscode window
+        const workspaceDir = this.getCurrentDir();
+
+        // Prioritize creating a team config in the local workspace if it exists even if a global config exists
+        // TODO: This behavior is only for the POC phase
+        ////////////////////////////////////////////////////////////////////////////////////////////////////////
+        if (workspaceDir !== undefined && !this.mProfilesCache.getTeamConfig().layerExists(workspaceDir)) {
+            await this.createZoweSchema(false);
+        } else {
+            await this.createZoweSchema(true);
+        }
```

Comment:
Since we call the same method in both branches, this could be simplified:
```suggestion
        const useProject = workspaceDir !== undefined && !this.mProfilesCache.getTeamConfig().layerExists(workspaceDir);
        await this.createZoweSchema(!useProject);
```

Reply:
I guess this was done in f4afdd854c5c59ada649042bf83785d5483e5853 ?